lists+dictionaries/colors.py

```diff
@@ -0,0 +1,11 @@
+input_name = input()
+color_name = input_name.split()
+input_code = input()
+color_code = input_code.split()
```

Comment:
Няма нужда от допълнителни променливи.
```suggestion
color_name = input().split()
color_code = input().split()
```